lib/delayed/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Delayed
-  VERSION = '1.1.0'
+  VERSION = '2.0.0.rc1'
```

Comment:
Not gonna cut a real release until I've convinced myself that this is roll-outable without speed bumps / gotchas.